app/helpers/login.ts

```diff
@@ -0,0 +1,24 @@
+import { Keypair } from "stellar-sdk";
+
+function getPublicKey(secretKey: string): string {
+  try {
+    return Keypair.fromSecret(secretKey).publicKey();
+  } catch (error) {
+    alert("Invalid secret key");
```

Comment:
This is not necessary. If you want to show the message, you can use a dependency to show that in the UI

Reply:
I have added a paragraph tag that displays an easy to see error message when the secret key is not correct and clears it when the entered text changes.
Would it be better to use a dependency like react-tostify?

Reply:
Yeah, that may work.